src/raml-cop.js

```diff
@@ -38,7 +39,7 @@ const validate = function (filename, options) {
 
       // Generic error
       if (!err.parserErrors) {
-        err.results = [{ src: filename, message: err.message }];
+        err.results = [{ src: filename, message: err.message, isWarning: err.isWarning }];
```

Comment:
raml-1-parser returns a common item for both errors and warnings, the only difference is this flag, so I only take this property to the raml-cop error item.